knowledge/animals/blue_foot_booby/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: 
+domain: animals
+seed_examples:
+- answer: |The natural breeding habitats of the blue-footed booby are the tropical and subtropical islands of the Pacific Ocean
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!